src/networkParams.ts

```diff
@@ -34,8 +34,8 @@ export class TransactionVersion {
     /**
      * Creates a TransactionVersion object with the VERSION setting for hash signing
      */
-    static withTxHashSignVersion(): TransactionVersion {
-        return new TransactionVersion(TRANSACTION_VERSION_TX_HASH_SIGN);
+    static withTxOptions(): TransactionVersion {
```

Comment:
Like above, we should document this change

Reply:
Documentation can be added here:

https://docs.elrond.com/sdk-and-tools/erdjs/erdjs-migration-guides/